signer/src/bitcoin/utxo.rs

```diff
@@ -451,6 +454,58 @@ impl<'a> Weighted for RequestRef<'a> {
     }
 }
 
+/// A struct for the transaction inputs and outputs associated with the
```

Comment:
Hm, maybe I'm just tired but I find this comment a little confusing, it mentions tx inputs and outputs but the struct contains references to withdrawal/deposit requests.. Or is it the associated txid's of said requests that you mean are the inputs/outputs?

Reply:
It's the associated txid's of the requests. Yeah let me clean this up.

Reply:
Updated: https://github.com/stacks-network/sbtc/pull/498/commits/f5808673b4d5d15a693cea003ee8440181c2526e